testsuite/features/reposync/srv_add_rocky8_repositories.feature

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@rhlike_minion
+Feature: Add the Rocky 8 distribution custom repositories
+  In order to use Rocky 8 channels with Red Hat "modules"
+  As a SUSE Manager administrator
+  I want to filter them out to remove the modules information
+
+  Scenario: Download the iso of Rocky 8 DVD and mount it on the server
+    When I mount as "rocky-8-iso" the ISO from "http://minima-mirror.mgr.suse.de/pub/rocky/8/isos/x86_64/Rocky-x86_64-dvd.iso" in the server
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Add a child channel for Rocky 8 DVD repositories
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Create Channel"
+    And I enter "Custom Channel for Rocky 8 DVD" as "Channel Name"
+    And I enter "rocky-8-iso" as "Channel Label"
+    And I select "RHEL8-Pool for x86_64" from "Parent Channel"
+    And I enter "Custom channel" as "Channel Summary"
+    And I click on "Create Channel"
+    Then I should see a "Channel Custom Channel for Rocky 8 DVD created" text
+
+  Scenario: Add the Rocky 8 Appstream DVD repository
+    When I follow the left menu "Software > Manage > Repositories"
+    And I follow "Create Repository"
+    And I enter "rocky-8-iso-appstream" as "label"
+    And I enter "http://127.0.0.1/rocky-8-iso/AppStream" as "url"
+    And I uncheck "metadataSigned"
+    And I click on "Create Repository"
+    Then I should see a "Repository created successfully" text
+
+  Scenario: Add the Rocky 8 BaseOS DVD repository
+    When I follow the left menu "Software > Manage > Repositories"
+    And I follow "Create Repository"
+    And I enter "rocky-8-iso-baseos" as "label"
+    And I enter "http://127.0.0.1/rocky-8-iso/BaseOS" as "url"
+    And I uncheck "metadataSigned"
+    And I click on "Create Repository"
+    Then I should see a "Repository created successfully" text
+
+  Scenario: Add both repositories to the custom channel for Rocky 8 DVD
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Custom Channel for Rocky 8 DVD"
+    And I follow "Repositories" in the content area
+    And I select the "rocky-8-iso-appstream" repo
+    And I select the "rocky-8-iso-baseos" repo
+    And I click on "Save Repositories"
+    Then I should see a "repository information was successfully updated" text
+
+  Scenario: Synchronize the repositories in the custom channel for Rocky 8 DVD
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Custom Channel for Rocky 8 DVD"
+    And I follow "Repositories" in the content area
+    And I follow "Sync"
+    # button is not available because a sync is already running from the scenario before this
+    # maybe we need a timeout
```

Comment:
Yeah, I guess we need to wait until a synchronization is done, but we should do it in the scenario that triggers it.

Reply:
Fixed with latest commit.